cache/volume_cache.go

```diff
@@ -180,6 +183,10 @@ func (c *VolumeCache) Commit() error {
 	return nil
 }
 
+func diffIDPath(basePath, diffID string) string {
+	return filepath.Join(basePath, strings.TrimPrefix(diffID, "sha256:")+".tar")
```

Comment:
`sha256:` in a file name causes problems on windows

Reply:
We considered that too. We weren't sure if it was worth the trouble/complexity for a one-time issue. Thoughts?

Reply:
I think it's necessary. Busting every cache will create problems for folks using CNB at scale.

Reply:
OK, will do :)